crates/sui-core/src/unit_tests/data/object_struct/sources/object_struct.move

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
nit, myabe give this a more informative name. "test_only_object_without_id" is long, but at least I don't have to open it to see what's going on. 

I think I am more concerned about someone not realizing this is just to provoke an error.

Reply:
Will do!